### ISSUE TITLE
LDAP ADD app is not complying the dark theme

### ISSUE THREAD

**Issue by ostasevych** [nextcloud/server#15657]
![ldap_ad_app_black_theme_issues](https://user-images.githubusercontent.com/5199030/58102424-ec166d80-7be9-11e9-8d7b-a32f70448c16.png)
When changed the NC theme to the dark one the colours of the fields of LDAP/ADD app remains the same, so the overall look is not comfortable.

### Steps to reproduce
1. Go to Personal Settings->Accessibility, select Dark theme, apply.
2. Go to Settings->LDAP/AD integration
3. You will see that the fields remain white.

### Expected behaviour
The colour of the fields should be inverted and look like other fields in the NC according to the selected theme.

### Actual behaviour
The fields of LDAP/ADD and the background of the frame is white and is not changing the colour dependently on the changed theme.

### Server configuration

**Operating system**: Centos 6.9

**Web server:** Apache

**Database:** MySQL

**PHP version:** 7.2

**Nextcloud version:** NC 6.0.1

**Updated from an older Nextcloud/ownCloud or fresh install:** updated from Nextcloud

**Where did you install Nextcloud from:** official nextcloud web-site

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
No errors have been found.
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>
```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>
![ldap_ad_app_black_theme_issues](https://user-images.githubusercontent.com/5199030/58102340-cdb07200-7be9-11e9-8af7-5ee07a1acf08.png)

**Comment by szaimen** [comment]
It looks like this is still valid.

**Comment by szaimen** [comment]
cc @nextcloud/javascript

**Comment by juliushaertl** [comment]
Mostly the jquery ui styling that the ldap app still uses is still containing hard coded color values which should rather be css variables:

https://github.com/nextcloud/server/blob/master/apps/user_ldap/css/settings.css#L200-L228

**Comment by szaimen** [comment]
Hi, please update to 24.0.8 or better 25.0.2 and report back if it fixes the issue. Thank you!
